app/routes/helper_functions.py

```diff
@@ -0,0 +1,14 @@
+from flask import abort, make_response
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"details": "Invalid Data"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"details": "Invalid Data"}, 404))
```

Comment:
404 HTTP status code is for "not found", so probably change this details message to something like `f"{cls.__name__} not found"`